released/packages/coq-ground/coq-ground.0.01.0/opam

```diff
@@ -0,0 +1,19 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Ground : A Usefull Extension to Coq's Standard Library"
+maintainer: "Joan Thibault <joan.thibault@ens-rennes.fr>"
+authors: "Joan Thibault <joan.thibault@ens-rennes.fr>"
+license: "LGPL-3.0-only with OCaml-LGPL-linking-exception"
+homepage: "https://gitlab.com/boreal-ldd/ground"
+bug-reports: "https://gitlab.com/boreal-ldd/ground/-/issues"
+depends: [
+  "coq" {>= "8.14.0"}
+]
+build: [make]
+install: [make "install"]
+remove: [make "uninstall"]
```

Comment:
```suggestion
build: [make "-j%{jobs}%"]
install: [make "install"]
```
I really recommend allowing parallel build. And opam doesn't need an uninstall, that is already taken care of.